WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeWebViewFragment.java

```diff
@@ -118,6 +121,17 @@ public void onResume() {
         setNavigationIcon(R.drawable.ic_close_white_24dp);
     }
 
+    private void clearCookies() {
```

Comment:
I tried that method (and many other similar methods) and unfortunately it doesn't seem to actually work. Setting a cookie doesn't actually overwrite it. It looks like people have had mixed results with different versions of Android.

Because of all of this I believe we should keep the deletion of all cookies.